README.md

```diff
@@ -0,0 +1,20 @@
+# example-sentence-extraction
+Module: Finding example sentences for keywords
+This module is responsible for fetching sentences containing the target user-inputed keyword, and by comparison with one another, select no more than 5 sentences which demonstrate the appropriate usage of the keyword.
+
+## Functional Design:
```

Comment:
It seems you are using pictures to display the functions, can you use code-formatted text instead? (Like in the example documentation). This will make it easier to modify later.